exchanges/bitstamp/bitstamp_websocket.go

```diff
@@ -163,47 +163,45 @@ func (b *Bitstamp) wsUpdateOrderbook(update websocketOrderBook, p currency.Pair,
 	}
 
 	var asks, bids []orderbook.Item
-	if len(update.Asks) > 0 {
-		for i := range update.Asks {
-			target, err := strconv.ParseFloat(update.Asks[i][0], 64)
-			if err != nil {
-				b.Websocket.DataHandler <- err
-				continue
-			}
-
-			amount, err := strconv.ParseFloat(update.Asks[i][1], 64)
-			if err != nil {
-				b.Websocket.DataHandler <- err
-				continue
-			}
+	for i := range update.Asks {
+		target, err := strconv.ParseFloat(update.Asks[i][0], 64)
+		if err != nil {
+			b.Websocket.DataHandler <- err
+			continue
+		}
 
-			asks = append(asks, orderbook.Item{Price: target, Amount: amount})
+		amount, err := strconv.ParseFloat(update.Asks[i][1], 64)
+		if err != nil {
+			b.Websocket.DataHandler <- err
+			continue
 		}
-	}
 
-	if len(update.Bids) > 0 {
-		for i := range update.Bids {
-			target, err := strconv.ParseFloat(update.Bids[i][0], 64)
-			if err != nil {
-				b.Websocket.DataHandler <- err
-				continue
-			}
+		asks = append(asks, orderbook.Item{Price: target, Amount: amount})
+	}
 
-			amount, err := strconv.ParseFloat(update.Bids[i][1], 64)
-			if err != nil {
-				b.Websocket.DataHandler <- err
-				continue
-			}
+	for i := range update.Bids {
+		target, err := strconv.ParseFloat(update.Bids[i][0], 64)
+		if err != nil {
+			b.Websocket.DataHandler <- err
+			continue
+		}
 
-			bids = append(bids, orderbook.Item{Price: target, Amount: amount})
+		amount, err := strconv.ParseFloat(update.Bids[i][1], 64)
+		if err != nil {
+			b.Websocket.DataHandler <- err
+			continue
 		}
+
+		bids = append(bids, orderbook.Item{Price: target, Amount: amount})
 	}
-	err := b.Websocket.Orderbook.Update(&wsorderbook.WebsocketOrderbookUpdate{
+
+	err := b.Websocket.Orderbook.LoadSnapshot(&orderbook.Base{
 		Bids:         bids,
 		Asks:         asks,
-		CurrencyPair: p,
-		UpdateID:     update.Timestamp,
+		Pair:         p,
+		LastUpdated:  time.Unix(update.Timestamp, 0),
```

Comment:
Since you're changing this from orderbook updates to snapshots, you can now remove the following from Bitstamp_wrapper
```
	b.Websocket.Orderbook.Setup(
		exch.WebsocketOrderbookBufferLimit,
		true,
		true,
		true,
		false,
		exch.Name)
```
As those properties are used for updates